concent_api/common/decorators.py

```diff
@@ -127,7 +128,7 @@ def wrapper(request, *args, **kwargs):
                     golem_message.task_id if 'task_id' in dir(golem_message) else None,
                     golem_message.subtask_id if 'subtask_id' in dir(golem_message) else None
                 )
-            except Http400 as exception:
+            except (Http400, ConcentValidationError) as exception:
```

Comment:
Is it still possible to catch `Http400` here ?

Reply:
No it's not, You'r right.